sdk/include/reactos/wine/port.h

```diff
@@ -381,4 +381,8 @@ ffs(int mask)
 
 #endif /* NO_LIBWINE_PORT */
 
+#if 0 /* Nonexistent on native NT. Add '#ifndef __REACTOS__' to related WINE code. */
+#define DLL_WINE_PREATTACH 8
+#endif
```

Comment:
Just remove that altogether. If someone adds a wine-synced module, compilation will fail if DLL_WINE_PREATTACH is present, and this will be an indication to do the ifndef REACTOS as it'll be done in the other modules, after your PR is merged.

Reply:
My suggestion was to have this as "explicit self-documentation".
But if it is not wanted...

Reply:
Fixed.